.github/workflows/ci.yml

```diff
@@ -1191,7 +1191,6 @@ jobs:
         else
           cmake -DCMAKE_BUILD_TYPE=${{ env.BUILD_TYPE }}     \
                 -DUSE_CLING=OFF                              \
```

Comment:
@faze-geek If use_repl has been made the default then we should be able to remove all the uses of use_cling=off, since it should be off by default.